### PR TITLE
Remove remaining Corporate Information Page code

### PR DESCRIPTION
**Pull request by jkempster34**
https://trello.com/c/lwjZfMY0

### Remove remaining Corporate Information Page code
The rendering of the Corporate Information Pages associated with Organisations
was moved to `government-frontend` some time ago.

This removes any remaining rendering code. The
`CorporateInformationPagesController` is now focussed entirely on Worldwide
Corporate Information Pages, which are also to be removed soon.

### Regenerate Brakeman ignore file
This:
- Removes any obsolete ignores
- Regenerates the ignores for the worldwide corporate information page because
some of the line numbers have changed.

---

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

Follow [these steps](https://guides.rubyonrails.org/upgrading_ruby_on_rails.html) if you are doing a Rails upgrade.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -60,13 +60,6 @@ def external_redirect(path_prefix, target)
   scope Whitehall.router_prefix, shallow_path: Whitehall.router_prefix do
     root to: redirect("/", prefix: ""), via: :get, as: :main_root
 
-    resources :organisations, only: [] do
-      # These aren't rendered but are coupled to Worldwide organisation corporate information pages
```
Comment (jkempster34) [inline config/routes.rb:64]: This comment says they are coupled, but I think it makes sense to start removing code where possible now that we are due to remove the worldwide CIPs too

File: app/controllers/corporate_information_pages_controller.rb [hunk 2]
```diff
@@ -47,9 +39,7 @@ def set_slimmer_headers_for_document
 
   def find_organisation
     @organisation =
-      if params.key?(:organisation_id)
-        Organisation.friendly.find(params[:organisation_id])
-      elsif params.key?(:worldwide_organisation_id)
+      if params.key?(:worldwide_organisation_id)
```
Comment (brucebolt) [inline app/controllers/corporate_information_pages_controller.rb:42]: Could the `if` be removed here as I'm assuming `WorldwideOrganisation.friendly.find(params[:worldwide_organisation_id])` would return an error if there's no `worldwide_organisation_id` anyway?

**Review by brucebolt — APPROVED** [review]
All looks good to me 👍.

Just left a small question about whether we need an if statement in the CIPs controller.

File: app/controllers/corporate_information_pages_controller.rb (hunk 2 above)
Comment (jkempster34) [reply app/controllers/corporate_information_pages_controller.rb]: Good point, thanks. Have removed!

**Pull request merged**